configuration/developing-community-health-applications.md

```diff
@@ -520,125 +529,122 @@ To build your targets into your app, you must compile them into app-settings, th
+|---|---|---|---|
+| `label` | `translation key` | Label shown with the field | yes |
+| `icon` | string | The name of the icon to display beside this field, as defined through the Configuration > Icons page. | no |
+| `value` | string | The value shown for the field | yes |
+| `width` | integer | The horizontal space for the field. Common values are 12 for full width, 6 for half width, or 3 for quarter width. Default 12 | no |
+| `translate` | boolean | Whether or not to translate the value. Defaults to false. | no |
+| `appliesIf` | boolean | True if the field should be shown. | no |
+| `appliesToType` | Array of contact types | Filters the contacts for which `appliesIf` will be evaluated. For example, `['person']` or `['clinic', 'health_center']`. | no |
 | `filter` | The display filter to apply to the value, eg: `{ value: '2005-10-09', filter: 'age' }` will render as "11 years". Common filters are: `age`, `phone`, `weeksPregnant`, `relativeDate`, `relativeDay`, `fullDate`, `simpleDate`, `simpleDateTime`, `lineage`, `resourceIcon`. For the complete list of filters, and more details on what each does, check out the code in [`medic/webapp/src/js/filters` dir](https://github.com/medic/medic/tree/master/webapp/src/js/filters). | no |
 
 ### Cards
-An array of cards to show below the summary on the profile page. Each card has its own header and arrays of fields.
+An array of cards to show below the summary on the profile page. Each card has its own header and array of fields.
 
 ![Pregnancy card](img/pregnancy-card.png)
 <!-- TODO: Add updated annotated screenshot of a profile card -->
 
-| property | description | required |
-|---|---|---|
-| `label` | Label on top of card | yes |
-| `appliesToType` | Type of contact where the card could show | no |
-| `appliesIf` | Function that determines if the card should show | no |
-| `fields` | Object for each field to show in the card | yes |
-| `fields.label` | Label shown with the field. Can be a translation key.  | yes |
-| `fields.value` | `function(r)`. The value shown for the field. Can be a property of the contact, eg `contact.date_of_birth`. | yes |
-| `fields.width` | The horizontal space for the field. Common values are 12 for full width, 6 for half width, or 3 for quarter width.| |
-| `fields.appliesIf` | A function that determines when the field should be shown. | no |
-| `fields.translate` | Whether or not to translate the value. Defaults to false. <!--TODO: VERIFY IMPLEMENTED--> | no |
-| `fields.context` |  The fields available in the value's translation. eg {} | no | 
-| `fields.icon` | The name of the icon to display beside this field, as defined through the Configuration > Icons page. | no |
-| `modifyContext` | function(ctx) | Used to modify or add values of the `context` | no |
+| property | type | description | required |
+|---|---|---|--|
+| `label` | `translation key` | Label on top of card | yes |
+| `fields` | Array of `Fields` | The content of the card | yes |
```

Comment:
I like the idea of consolidating to a common Field type. 

Are we sure that all fields in `Fields` are valid here? Eg `appliesToType`

Also, some available in this schema are not in `Fields` eg `fields.context`.

If we can reconcile those difference this makes a lot of sense.

Reply:
You're right. Some weird stuff happening...

Reply:
Okay - reverted and corrected.